lms/templates/provider/authorize.html

```diff
@@ -0,0 +1,55 @@
+{% extends "main_django.html" %}
+
+{% load scope i18n %}
+
+{% block bodyclass %}oauth2{% endblock %}
+
+{% block body %}
+<div class="authorization-confirmation">
+  {% if not error %}
+  {% blocktrans with application_name=client.name %}
+  <p><strong>{{ application_name }}</strong> would like to access your data with the following permissions:</p>
+  {% endblocktrans %}
+  <ul>
+    {% for permission in oauth_data.scope|scopes %}
+    <li>
+      {% if permission == "openid" %}
+        {% trans "Read your user ID" %}
+      {% elif permission == "profile" %}
+        {% trans "Read your user profile" %}
+      {% elif permission == "email" %}
+        {% trans "Read your email address" %}
+      {% elif permission == "course_staff" %}
+        {% trans "Read the list of courses in which you are a staff member." %}
+      {% elif permission == "course_instructor" %}
+        {% trans "Read the list of courses in which you are an instructor." %}
+      {% elif permission == "permissions" %}
+        {% trans "To see if you are a global staff user" %}
+      {% else %}
+        {% blocktrans %}Manage your data: {{ permission }}{% endblocktrans %}
+      {% endif %}
+    </li>
+    {% endfor %}
+  </ul>
+  <form method="post" action="{% url "oauth2:authorize" %}">
+    {% csrf_token %}
+    {{ form.errors }}
+    {{ form.non_field_errors }}
+    <fieldset>
+      <div style="display: none;">
+        <select type="select" name="scope" multiple="multiple">
+          {% for scope in oauth_data.scope|scopes %}
+          <option value="{{ scope }}" selected="selected">{{ scope }}</option>
+          {% endfor %}
+        </select>
+      </div>
+      <input type="submit" class="btn login large danger" value="Cancel" />
```

Comment:
Based on the underlying library.. yes. This should be a submit button. See here:

https://github.com/edx/django-oauth2-provider/blob/edx/provider/views.py#L397